app/static/script.css

```diff
@@ -0,0 +1,1239 @@
+  color: #7f8cf7;
+  padding: 10px 0;
+}
+
+.icons {
+  display: flex;
+  max-width: 210px;
+  align-content: center;
+  justify-content: space-evenly;
+  gap: 20px;
+}
+
+.fa-brands,
+.fa-solid {
+  color: #505f79;
+  font-size: 24px;
+  transition: transform 0.5s;
+  margin: 0 auto;
+}
+
+.fa-brands:hover,
+.fa-solid:hover {
+  color: #2230d2;
+  transform: scale(1.2);
+  transform: scale(1.5);
+}
+
+#logo-text {
+  animation: rotate 1s ease-out;
+}
+
+/* +++++++++++++++++++++PORTFOLIO CARD SECTION++++++++++++++++++ */
+.about-lang-frame > li {
+  padding: 10px;
+}
+
+.buttons > li {
+  background: #fff;
+  border: 1px solid #ebecf0;
+  box-shadow: 0 48px 48px rgb(37 47 137 / 8%);
+  display: flex;
+  align-items: center;
+  padding: 8px 25px;
+  gap: 10px;
+  width: 142px;
+  height: 40px;
+
+  /* border: 1px solid #6070ff; */
+  border-radius: 8px;
+  font-weight: 800;
+  font-size: 15px;
+  line-height: 20px;
+  color: #396df2;
+}
+
+.works-section {
+  height: 496px;
+  width: 1156px;
+  border-radius: 24px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  display: flex;
+  gap: 50px;
+  align-items: center;
+  margin: 0 auto;
+  margin-bottom: 100px;
+}
+
+.work-img {
+  height: 448px;
+  width: 544px;
+  border-radius: 8px;
+  margin: 5px;
+}
+
+.work-title {
+  margin-top: -50px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #091e42;
+  padding: 0 30px;
+}
+
+.tech-list > li {
+  align-items: center;
+  padding: 4px 12px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+  text-align: center;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 26px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+.works-badges {
+  display: flex;
+  gap: 25px;
+  list-style-type: none;
+  margin: 5px;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 24px;
+  color: #6b778c;
+  padding: 5px 30px;
+}
+
+/* ----Select the first list element--- */
+.works-badges > li:first-of-type {
+  height: 24px;
+  text-transform: uppercase;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.works-para {
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  width: 496px;
+  height: 50px;
+  padding: 5px 30px;
+}
+
+.tech-list {
+  display: flex;
+  gap: 10px;
+  padding: 5px 40px;
+  list-style-type: none;
+  margin-bottom: 20px;
+}
+
+button {
+  padding: 12px;
+  width: 144px;
+  height: 48px;
+  border-radius: 8px;
+  border: none;
+  font-weight: 600;
+  font-size: 16px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  margin-left: 30px;
+}
+
+/* +++++++++++++++++++++++++++++Modal detail++++++++++++++++++++++++++ */
+#my-modal {
+  width: 1156px;
+  height: 100%;
+  display: none;
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  overflow: auto;
+  background-color: #fff;
+  border: 1px solid #ebecf0;
+  box-shadow: 0 48px 48px rgb(37 47 137 / 8%);
+  border-radius: 16px;
```

Comment:
- [ ] [REQUIRED] Please correct the `position` of the `modal`. In the picture below I have shown them the `difference`!

![image](https://user-images.githubusercontent.com/63915024/190514825-097f0dc0-748a-43e9-a2dd-a02b899472ed.png)

Reply:
Hello @mukhammadrizooff 
Do you have any leads on how I can solve this? 
Thanks.